.gitignore

```
@@ -1,16 +1,12 @@
 # Config file (containing api keys and such)
 .config
 
-# Jupyter notebook
-*.ipynb
```

I would prefer generally prohibiting notebooks and just addings pecific ones that we want to keep.

```suggestion
*.ipynb
!tutorial_*.ipynb
```